product_docs/docs/edbcloud/beta/administering_cluster/01_portal_access.mdx

```diff
@@ -0,0 +1,111 @@
+
+* Each organization will have three default `roles` available:
+  * _owner_
+  * _reader_
+  * _contributor_
+* These roles can be edited by changing their name or description.
+
+### Permissions
+
+Permissions are generally represented in the format `action:object` where `action` represents an operation that can be performed and `object` represents a category of portal functionality.
+
+The available `actions` are: _create_, _read_, _update_, _delete_
+
+The available `objects` are: _backups_, _billing_, _databases_, _events_, _permissions_, _roles_, _tickets_, _users_,  _versions_
+
+### Permissions by Role
+
+The following are the default permission by role:
+
+| Role        | Action |backups | billing | databases | events | roles | permissions | tickets | users | versions |
+|-------------|--------|--------|---------|-----------|--------|-------|-------------|---------|-------|----------|
+| owner       | create |   x    |         |     x     |        |    x  |       x     |     x   |   x   |    x     |
+|             | read   |   x    |    x    |     x     |   x    |    x  |       x     |     x   |   x   |    x     |
+|             | update |   x    |         |     x     |        |    x  |       x     |     x   |   x   |    x     |
+|             | delete |   x    |         |     x     |        |    x  |       x     |     x   |   x   |    x     |
+| contributor | create |   x    |         |     x     |        |       |             |     x   |       |          |
+|             | read   |   x    |    x    |     x     |   x    |    x  |       x     |     x   |   x   |    x     |
+|             | update |   x    |         |     x     |        |       |             |     x   |       |          |
+|             | delete |   x    |         |     x     |        |       |             |     x   |       |          |
+| reader      | create |        |         |           |        |       |             |         |       |          |
+|             | read   |   x    |    x    |     x     |   x    |    x  |       x     |     x   |   x   |    x     |
+|             | update |        |         |           |        |       |             |         |       |          |
+|             | delete |        |         |           |        |       |             |         |       |          |
+
+
+### Editing Roles
+
+To edit roles:
+1. Navigate to Admin > Roles.
+
+3. Select the edit icon for the role in the list.
+
+#### Changing Role Name
+
+To change the name or description of the role:
+1. Select the **Settings** tab.
+
+2. Edit **Name** or **Description**.
+3. Press **Save**.
+
+#### Changing Role Permissions
+
+To change permissions associated with the role:
+1. Select **Permissions** tab.
+
+2. Select the **Change Permissions** button on the top right.
+3. Select the list of permissions you want to be associated with the role.
+4. Press **Submit**
+
+!!! Note:
```

Comment:
This is rendering badly, for some reason - the "Note" text is being removed. The one on L92, without the colon, renders fine.

<img width="826" alt="Screen Shot 2021-08-11 at 10 21 37 AM" src="https://user-images.githubusercontent.com/79652654/129074606-ab6955ef-c48b-4061-a16d-0b2e58b0fd01.png">